repl/src/main/scala/ammonite/repl/Storage.scala

```diff
@@ -90,6 +133,63 @@ object Storage{
       }
     }
 
+    def classFilesListSave(pkg: String,
+                           wrapper: String,
+                           dataList: Seq[(String, String)],
+                           imports: Imports,
+                           tag: String,
+                           importTreesList: Seq[ImportTree]): Unit = {
+      val dir = pkg.replace("/", "$div") + "." + wrapper.replace("/", "$div") + "0"
```

Comment:
As discussed, we should not randomly prefix things with `0`: these should go in a separate folder from the `cache/compile/`, e.g. `cache/scriptLevel/`